### PR TITLE
Docker cross images

### PR DESCRIPTION
**Pull request by sagudev**

### CODE REVIEW

**Comment by sagudev** [comment]
Working docker images are used here: https://github.com/sagudev/mozjs/actions/runs/6186951444/job/16795707297 and they are working

File: docker/Dockerfile.cross-aarch64-unknown-linux-gnu
Comment (mukilan) [inline docker/Dockerfile.cross-aarch64-unknown-linux-gnu:1]: GitHub's file viewer doesn't seem to recognize this as a Dockerfile for syntax highlighting. Maybe calling it `<variant>.dockerfile` instead will help?

**Comment by mukilan** [comment]
Nice!

File: docker/Dockerfile.cross-aarch64-unknown-linux-gnu
Comment (sagudev) [reply docker/Dockerfile.cross-aarch64-unknown-linux-gnu:1]: Good catch.

**Review by mukilan — APPROVED** [review]
Thanks!

**Pull request merged**